.gitignore

```diff
@@ -109,7 +109,7 @@ celerybeat.pid
 *.sage.py
 
 # Environments
-.env
+# .env
```

Comment:
comment or delete? and would we rather whitelist `.devcontainer/.env` and still ignore all other `.env` files?

Reply:
oops, right. Left it to ignore .env in the root folder.